src/components/Card/GenerativeTokenCard.tsx

```diff
@@ -30,7 +31,7 @@ export function GenerativeTokenCard({
   lockedUntil,
 }: Props) {
   const url = getGenerativeTokenUrl(token)
-
+  const isBetaToken = token.objkts?.[0]?.version == 0
```

Comment:
this is a workaround for the currently query. probably there is a smarter way to add the version to the tokens.